README.md

```diff
@@ -1,8 +1,6 @@
 # ActiveVersioning
 
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/active_versioning`. To experiment with that code, run `bin/console` for an interactive prompt.
-
-TODO: Delete this and the text above, and describe your gem
+ActiveVersioning provides out-of-the-box versioning functionality in Rails.  ActiveVersioning serializes attributes when records are saved and allows for version and draft management.
```

Comment:
Looks like some of the code examples in the README show using `current_admin`. Is this thing expected to be an enhancement only when using Active Admin? If so, might be worth noting in the intro.